bots/elasticsearch/tasks.py

```diff
@@ -18,6 +21,22 @@ def safe_substr(value, length=32000):
         return str(value)[:length]
     return None
 
+def add_suggest(obj):
+    input = obj['name']
+    if input:
+        # Elastic's completion suggester only looks at prefixes. If we give each
```

Comment:
For any combination of those two settings, the completion suggester still only matches prefixes. The example in the docs where `b` matches 'The Beatles' when `preserve_position_increments` is `false` only applies with an analyzer that removes stop words, so the analyzed string would be `beatles`.

The edge n-gram autocomplete we had before is a more natural solution to matching words in the middle of the string, but I think we'll still get the speed benefit of the completion suggester, even with a bunch of inputs per object.